cmd/root.go

```diff
@@ -165,21 +166,42 @@ func functionWithOverrides(root, namespace, image string) (f faas.Function, err
 
 // deriveName returns the explicit value (if provided) or attempts to derive
 // from the given path.  Path is defaulted to current working directory, where
-// a function configuration, if it exists and contains a name, is used.  Lastly
-// derivation using the path us used.
+// a Function configuration, if it exists and contains a name, is used. 
 func deriveName(explicitName string, path string) string {
```

Comment:
I wonder if this function could be better named as it no longer really derives the name, but instead just pulls it from the configuration file or returns the passed value.

Reply:
Any suggestion?

Reply:
> Any suggestion?

I've been sitting here for 5 minutes trying to come up with a good name. I see why you asked. The best I've come up with is `providedOrConfiguredName(string) string`. But meh.

Reply:
Hehe, yeah 😄 I don't like the name either, so I am happy to change it to anything else, that makes sense for everyone